sphinx/source/docs/releases/0.12.6.rst

```diff
@@ -38,10 +38,14 @@ Old                            New
 ``mappers/CategoricalMapper``  ``scales/CategoricalScale``
 ============================== ==============================
 
-Previously, these classes were internal implementation details. These changes
-are not expected to impact any user code.
-
-Additionally, these new scales models are now accessible from the Python.
+Previously, these Mapper classes were internal implementation details. The
+new Scale classes are first-class Bokeh models and are accessible from
+Python.
+
+There is a new Plot validation check to ensure that Scales and Ranges on a
+dimension are compatible. Users employing a ``FactorRange`` in the
+``bokeh.plotting`` API will have to specify ``CategoricalScale`` on the same
```

Comment:
meant ``bokeh.models`` here? Things are handled automagically in ``bokeh.plotting`` for most cases? I think its worth showing an example line or two of code here to demonstrate exactly what users will need to change, and possibly a link to a complete example that does this.